stdlib/public/core/Integers.swift.gyb

```diff
@@ -240,52 +240,73 @@ def operatorComment(operator, fixedWidth):
   ///   - rhs: The value to divide `lhs` by. `rhs` must not be zero.
 """,
         '&+': """\
-  /// Returns the sum of the two given values, discarding any overflow.
+  /// Returns the sum of the two given values, wrapping any overflow.
```

Comment:
This is better than it was, but it still isn't great; it's not the the overflow that wraps, but rather the result in case of overflow. We should probably also make the point somewhere that the result is "correct" mod 2**N when working with `IntN`, though I'm not sure how long we want these comments to get =)